src/v2/api/index.md

```diff
@@ -1346,6 +1346,23 @@ type: api
     }
   })
   ```
+    
+  <p class="tip">As a rule of thumb, make sure you don't name them with the prefixes `$` or `_`, as the Vue api/internal properties or misuse or overwrite the existing ones.</p>
+  
+  Note that they [won't be reactive](https://github.com/vuejs/vue/issues/1988#issuecomment-355980148). If you need reactivity, you need to initialize the properties as usual: within [`data`](#data) (or [`computed` properties](#computed)), passing [`props`](#props) or for more advanced use cases [`injecting`](#provide-inject) reactive objects. Bear in mind that the recommended way of adding non-reactive properties to the current instance is using the `created()` hook, as described [here](https://github.com/vuejs/vue/issues/1988#issuecomment-163013818): 
```

Comment:
Some feedback:

* The GitHub issue links are not necessary.
* The part of "If you need reactivity…" is not necessary. 
* AFAIK using `created()` is _not_ exactly a more recommended method, more like an alternative. Note that Evan's comment was 4 years ago and didn't mention anything about recommendation.